build/root/.kazelcfg.json

```diff
@@ -2,6 +2,7 @@
 	"GoPrefix": "k8s.io/kubernetes",
 	"SkippedPaths": [
 		"^_.*",
+		"/_",
```

Comment:
do we need the `.*` or not?

Reply:
Not really. `.*` will just result in a bigger match, but all we care about if there's a match or not